src/app/Main/Header/Header.component.tsx

```diff
@@ -134,9 +125,7 @@ export function HeaderComponent(): JSX.Element {
                 </div>
             </div>
 
-            <div css={subheaderStyle(queryOpen)}>
-                <HeaderExportContainerComponent />
-            </div>
+            {queryOpen && <HeaderExportContainerComponent />}
```

Comment:
Good, generally avoid display:none for putting together the UI. You use the react framework and it provides a flexible and fast way for rendering components depending on the apps state. So by using display:none you actually bypass the framework.

Reply:
Commit: "Remove "display: none" situations "